h2o-bindings/bin/custom/R/gen_stackedensemble.py

```diff
@@ -57,6 +58,14 @@ def update_param(name, param):
     model@parameters$metalearner_params <- list(fromJSON(model@parameters$metalearner_params))[[1]] #Need the `[[ ]]` to avoid a nested list
 }
 model@model <- .h2o.fill_stackedensemble(model@model, model@parameters, model@allparams)
+
```

Comment:
now you see why it's better to trust the backend in your previous ticket about `base_models` ;)
I didn't even think about current ticket at that time.